app/controllers/ember-community-survey-2017.js

```diff
@@ -531,232 +451,195 @@ const recommendingEmber = new VerticalBarChart({
+    ],
+    title:
+      'Which of these best describes your use of Ember in production applications?',
   },
-  data: [
+
+  rawData: [
     {
-      name: '2016',
+      label: '2016',
       color: darkGrayColor,
-      data: [15.11, 26.2, 34.2, 39.6, 8.2],
+      values: [15.11, 26.2, 34.2, 39.6, 8.2],
     },
     {
-      name: '2017',
+      label: '2017',
       color: emberOrange,
-      data: [12.24, 24.42, 36.85, 42.4, 9.27],
+      values: [12.24, 24.42, 36.85, 42.4, 9.27],
     },
   ],
-};
+}).highchartsOptions;
 
-const whenStarting = {
-  options: {
-    chart: {
-      type: 'column',
-    },
-    title: {
-      text: 'When starting your most recent job...',
-    },
-    subtitle: {
-      text: '',
-    },
-    xAxis: {
-      type: 'category',
-    },
-    yAxis: {
-      title: {
-        text: 'Percent',
-      },
-    },
+const whenStarting = new VerticalBarChart({
+  chart: {
+    categories: [
+      'My employer was not using Ember',
+      'My employer was already using Ember',
+      'Does not apply (e.g. I am a student)',
+    ],
+    title: 'When starting your most recent job...',
   },
-  data: [
+
+  rawData: [
     {
-      name: '2017',
+      label: '2017',
       color: emberOrange,
-      data: [
-        { name: 'My employer was not using Ember', y: 53.63 },
-        { name: 'My employer was already using Ember', y: 38.74 },
-        { name: 'Does not apply (e.g. I am a student)', y: 6.56 },
-      ],
+      values: [53.63, 38.74, 6.56],
     },
   ],
-};
+}).highchartsOptions;
 
-const preExistingKnowledge = {
-  options: {
-    chart: {
-      type: 'column',
-    },
-    title: {
-      text:
-        'When getting hired to your most recent job, your pre-existing knowledge of Ember was...',
-    },
-    subtitle: {
-      text: '',
-    },
-    xAxis: {
-      type: 'category',
-    },
-    yAxis: {
-      title: {
-        text: 'Percent',
-      },
-    },
+const preExistingKnowledge = new VerticalBarChart({
+  chart: {
+    categories: [
+      'I did not know Ember',
+      'Unimportant',
+      'Very important',
+      'Important',
+    ],
+    title:
+      'When getting hired to your most recent job, your pre-existing knowledge of Ember was...',
   },
-  data: [
+
+  rawData: [
     {
-      name: '2017',
+      label: '2017',
       color: emberOrange,
-      data: [
-        { name: 'I did not know Ember', y: 47.13 },
-        { name: 'Unimportant', y: 25.49 },
-        { name: 'Very important', y: 12.87 },
-        { name: 'Important', y: 12.87 },
-      ],
+      values: [47.13, 25.49, 12.87, 12.87],
     },
   ],
-};
+}).highchartsOptions;
 
-const demographics = {
-  options: {
-    chart: {
-      type: 'column',
-    },
-    title: {
-      text: 'In which region do you live?',
-    },
-    subtitle: {
-      text: '',
-    },
-    xAxis: {
-      type: 'category',
-    },
-    yAxis: {
-      title: {
-        text: 'Percent',
-      },
-    },
-    drilldown: {
```

Comment:
Good catch. This is the first time I'm encountering `drilldown` property.

I checked that, in the production site, the drilldown doesn't work (clicking on North America bar did nothing) and doesn't also error.

Since this is the only place is `drilldown` is used (the `drilldown` data in 2018 is a copy-paste of that in 2017), I'll remove it in a separate pull request.